tests/test_data/PO files/bom/strings.po

```diff
@@ -0,0 +1,27 @@
+﻿# Kodi Media Center language file
```

Comment:
please remove or replace the space for all the files located at `tests/test_data/PO files`.

Reply:
The spaces are replaced with `_`